### PR TITLE
Removes Broken Wallpapers

### PR DESCRIPTION
**Pull request by Aniq55**
Fixes: #293

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

File: uiplib/gui/gallery.py [hunk 1]
```diff
@@ -42,7 +44,12 @@ def set_image(self, imagePath):
                                 label="Blur", showvalue=0)
             self.slider.pack()
         self.image = UipImage(imagePath)
-        self.show_image(self.image)
+        im = Image.open(imagePath)
+        try:
+            im.load()
+            self.show_image(self.image)
+        except IOError as e:
```
Comment (nemaniarjun) [inline uiplib/gui/gallery.py]: You don't need to add as `e`

Comment (nemaniarjun) [inline uiplib/gui/gallery.py]: Remove the `as e`

Comment (Aniq55) [inline uiplib/gui/gallery.py]: ok, will change that later, its of no harm though

File: uiplib/gui/gallery.py [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 from tkinter import *
 from PIL import ImageTk
+from PIL import Image
```
Comment (nemaniarjun) [inline uiplib/gui/gallery.py]: You can import both in one line

File: uiplib/gui/gallery.py [hunk 3]
```diff
@@ -2,7 +2,9 @@
...
 from uiplib.uipImage import UipImage
+import os
```
Comment (nemaniarjun) [inline uiplib/gui/gallery.py]: You need to see PEP8 standard for importing.

File: uiplib/gui/gallery.py [hunk 4]
```diff
@@ -42,7 +44,12 @@ def set_image(self, imagePath):
                                 label="Blur", showvalue=0)
             self.slider.pack()
         self.image = UipImage(imagePath)
-        self.show_image(self.image)
+        im = Image.open(imagePath)
+        try:
+            im.load()
```
Comment (nemaniarjun) [inline uiplib/gui/gallery.py]: Are we sure this is how we want to do this? Maybe add this to UIP image object? Consult @abhsag24

Comment (abhsag24) [inline uiplib/gui/gallery.py]: not sure myself, yes the UIP image class should be used but even then i'm not sure we should check here

File: uiplib/gui/gallery.py (hunk 3 above)
Comment (Aniq55) [inline uiplib/gui/gallery.py]: It did not give any linting error though

Comment (abhsag24) [inline uiplib/gui/gallery.py]: @Aniq55 yes but we do have standard imports and it'll be better if we maintain them, can you check the wikis? i think we have it there.

File: uiplib/gui/gallery.py (hunk 4 above)
Comment (Aniq55) [inline uiplib/gui/gallery.py]: @abhsag24 If the approach needs to be different i.e. changes are to be made in a different file, then I think it is better if the PR is closed.

Comment (abhsag24) [inline uiplib/gui/gallery.py]: @Aniq55 why do you think so?

Comment (abhsag24) [inline uiplib/gui/gallery.py]: btw it would be better if you use our image class and make changes to that.

Comment (nkprince007) [inline uiplib/gui/gallery.py]: Yeah, it's better using the wrapper @Aniq55. Don't close the PR, BTW.

Comment (Aniq55) [inline uiplib/gui/gallery.py]: Alright! @nkprince007 @abhsag24

**Comment by nkprince007** [comment]
Closing as it's been unattended for too long. Reopen it if needed.

**Pull request closed without merging**